src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,45 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
-);
+export function MovieCard({ imgUrl, title, description, imdbUrl }) {
+  return (
+    <div className="card">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            src={imgUrl}
+            alt="Film logo"
+          />
+        </figure>
+      </div>
+
+      <div className="card-content">
+        <div className="media">
+          <div className="media-left">
+            <figure className="image is-48x48">
+              <img src="/images/imdb-logo.jpeg" alt="imdb" />
+            </figure>
+          </div>
+
+          <div className="media-content">
+            <p className="title is-8">{title}</p>
+          </div>
+        </div>
+
+        <div className="content">
+          {description}
+          <br />
+          <a href={imdbUrl}>IMDB</a>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+MovieCard.propTypes = {
+  imgUrl: PropTypes.string.isRequired,
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+  imdbUrl: PropTypes.string.isRequired,
```

Comment:
In the task, description must be without isRequired property. Instead, you need to add description to the defaultProps